apps/judicial-system/xrd-api/src/app/app.dto.ts

```diff
@@ -11,9 +17,11 @@ export class CreateCaseDto {
   readonly type!: CaseType
 
   @IsNotEmpty()
-  @IsString()
+  @IsArray()
+  @ArrayMinSize(1)
+  @IsString({ each: true })
   @ApiProperty()
-  readonly policeCaseNumber!: string
+  readonly policeCaseNumbers!: string[]
```

Comment:
This cannot be changed at this time as the change will break our contract with LÖKE.

Reply:
True, forgot about that. I will update so that the endpoint still accepts a single number and call it internally with a list.